> just pass `RUSTFLAGS+=" --cfg tokio_unstable"`. it's probably set upstream via config.toml, but environment variable is preferable over config

Thanks! But we get next error:
```console
   Compiling datafusion-functions-window v53.1.0
error[E0308]: mismatched types
   --> crates\ecstore\src\rpc\remote_disk.rs:900:36
    |
900 |                     file_info_bin: file_info_bin.clone(),
    |                                    ^^^^^^^^^^^^^^^^^^^^^ expected `Bytes`, found `Vec<u8>`
    |
    = note: expected struct `bytes::Bytes`
               found struct `Vec<u8>`
help: call `Into::into` on this expression to convert `Vec<u8>` into `bytes::Bytes`
    |
900 |                     file_info_bin: file_info_bin.clone().into(),
    |                                                         +++++++

error[E0308]: mismatched types
   --> crates\ecstore\src\rpc\remote_disk.rs:966:36
    |
966 |                     file_info_bin: file_info_bin.clone(),
    |                                    ^^^^^^^^^^^^^^^^^^^^^ expected `Bytes`, found `Vec<u8>`
    |
    = note: expected struct `bytes::Bytes`
               found struct `Vec<u8>`
help: call `Into::into` on this expression to convert `Vec<u8>` into `bytes::Bytes`
    |
966 |                     file_info_bin: file_info_bin.clone().into(),
    |                                                         +++++++

error[E0308]: mismatched types
   --> crates\ecstore\src\rpc\remote_disk.rs:967:31
    |
967 |                     opts_bin: opts_bin.clone(),
    |                               ^^^^^^^^^^^^^^^^ expected `Bytes`, found `Vec<u8>`
    |
    = note: expected struct `bytes::Bytes`
               found struct `Vec<u8>`
help: call `Into::into` on this expression to convert `Vec<u8>` into `bytes::Bytes`
    |
967 |                     opts_bin: opts_bin.clone().into(),
    |                                               +++++++

error[E0308]: mismatched types
    --> crates\ecstore\src\rpc\remote_disk.rs:1009:31
     |
1009 |                     opts_bin: opts_bin.clone(),
     |                               ^^^^^^^^^^^^^^^^ expected `Bytes`, found `Vec<u8>`
     |
     = note: expected struct `bytes::Bytes`
                found struct `Vec<u8>`
help: call `Into::into` on this expression to convert `Vec<u8>` into `bytes::Bytes`
     |
1009 |                     opts_bin: opts_bin.clone().into(),
     |                                               +++++++

error[E0308]: mismatched types
    --> crates\ecstore\src\rpc\remote_disk.rs:1483:21
     |
1483 |                     read_multiple_req_bin,
     |                     ^^^^^^^^^^^^^^^^^^^^^ expected `Bytes`, found `Vec<u8>`
     |
     = note: expected struct `bytes::Bytes`
                found struct `Vec<u8>`
help: call `Into::into` on this expression to convert `Vec<u8>` into `bytes::Bytes`
     |
1483 |                     read_multiple_req_bin: read_multiple_req_bin.into(),
     |                     ++++++++++++++++++++++                      +++++++

   Compiling datafusion-optimizer v53.1.0
warning: unused variable: `start`
    --> crates\ecstore\src\disk\local.rs:1924:13
     |
1924 |         let start = Instant::now();
     |             ^^^^^ help: if this is intentional, prefix it with an underscore: `_start`
     |
     = note: `#[warn(unused_variables)]` (part of `#[warn(unused)]`) on by default

   Compiling reqwest v0.12.28
   Compiling datafusion-functions-table v53.1.0
   Compiling datafusion-catalog-listing v53.1.0
   Compiling snafu-derive v0.9.0
   Compiling quick-xml v0.37.5
   Compiling crypto-bigint v0.7.3
   Compiling serde-value v0.7.0
   Compiling arc-swap v1.9.1
   Compiling nom v8.0.0
For more information about this error, try `rustc --explain E0308`.
warning: `rustfs-ecstore` (lib) generated 1 warning
error: could not compile `rustfs-ecstore` (lib) due to 5 previous errors; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
==> ERROR: A failure occurred in build().
    Aborting...
```

--- a/mingw-w64-rustfs/PKGBUILD
+++ b/mingw-w64-rustfs/PKGBUILD
@@ -3,9 +3,9 @@
 _realname=rustfs
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-_pkgver=1.0.0-alpha.89
+_pkgver=1.0.0-alpha.98
 pkgver=${_pkgver//-/_}
-_consolever=0.1.5
+_consolever=0.1.7
 pkgrel=1
 pkgdesc="High-performance, memory-safe distributed object storage (mingw-w64)"
 arch=('any')
@@ -36,8 +36,8 @@ source=(
 )
 noextract=("rustfs-${_pkgver}.tar.gz"
            "rustfs-console-v${_consolever}.zip")
-sha256sums=('bd4ba41dd75ef9ae133f053cb07a962d3a021021428d91047e02b6047d4b7e4f'
-            '3fb4e699b905735982d20b4d42d1bcda48f43614ce1dfd0ccb060e69b93802ca')
+sha256sums=('32586e9bd7f9970f4ec2f07a42ed61d7294defe5596f7698a515f3bfd1cdcfef'
+            'e9c11e4a45ddbc7fb094480fa72518d919cfb86a506ce4b7795549531fb958ab')
 
 prepare() {
   echo "Extracting ${_realname}-${_pkgver}.tar.gz ..."
@@ -63,8 +63,8 @@ build() {
   cd "${_realname}-${_pkgver}"
 
   export RUSTUP_TOOLCHAIN=stable-${RUST_CHOST}
-  # For more info see https://github.com/msys2/MINGW-packages/pull/28451
-  export AWS_LC_SYS_CFLAGS=-O0
+  # For more info see https://github.com/msys2/MINGW-packages/pull/29061
+  export RUSTFLAGS="${RUSTFLAGS} --cfg tokio_unstable"
 
   # Generate protobuf/flatbuffers code (uses protoc/flatc from distro)
   cargo run --release --frozen --bin gproto

